installbuilder/datafiles/Base_DSC.data

```diff
@@ -160,28 +160,13 @@ fi
 #endif
 
 %Preinstall_200
-tmpfile=`mktemp`
 echo "Checking for ctypes python module..."
-cat <<EOF > $tmpfile
-#!/usr/bin/python
-import ctypes
-EOF
-
-chmod u+x $tmpfile
-$tmpfile 1> /dev/null 2> /dev/null
-tmpfile_ctypes=$?
-rm $tmpfile
-
 python -c "import ctypes" 1> /dev/null 2> /dev/null
 cmdline_ctypes=$?
```

Comment:
With your changes, this line can be removed altogether and just check `$? -ne 0` in the if.